src/main/java/org/kainos/ea/api/BandService.java

```diff
@@ -28,11 +33,15 @@ public List<Band> getBands() throws FailedToGetBandsException {
 
     }
 
-    public Band getBandByID(int id) throws FailedToGetBandException {
+    public Band getBandByID(int id) throws FailedToGetBandException, BandDoesNotExistException {
         Band band = new Band();
```

Comment:
This could have unintended consequences. What if this ever got returned from this method?